tests/integration/integration.spec.js

```diff
@@ -657,4 +657,71 @@ describe('FVL integration', () => {
     expect(button.element.disabled).toBe(false)
     expect(wrapper.find('#error').text()).toBe('')
   })
+
+  it('validates fields with array in array schema', async () => {
+    const SchemaWithValidation = SchemaFormFactory([veeValidatePlugin()])
+    const schema = {
+      user: {
+        component: SchemaWithValidation,
+        model: 'subform',
+        schema: [
+          {
+            model: 'email',
+            label: 'Email',
+            component: FormText,
+            validations: yup.string().required(REQUIRED_MESSAGE)
+          }, [
+            {
+              model: 'firstName',
+              label: 'First Name',
+              component: FormText,
+              validations: yup.string().required(REQUIRED_MESSAGE)
+            }, {
+              model: 'lastName',
+              label: 'Last Name',
+              component: FormText,
+              validations: yup.string().required(REQUIRED_MESSAGE)
+            },
+          ]
+        ]
+      }
+    }
+
+    const wrapper = mount({
+      template: `
+        <SchemaWithValidation :schema="schema" />
+      `,
+      components: {
+        SchemaWithValidation
+      },
+      setup () {
+        const formData = ref({})
+        useSchemaForm(formData)
+
+        return {
+          schema
+        }
+      }
+    })
+
+    const inputs = wrapper.findAllComponents(FormText)
+    const errors = wrapper.findAll('span')
+    const form = wrapper.find('form')
+
+    inputs[0].setValue('test@gmail.com')
+    await flushPromises()
+    expect(errors[0].text()).toBe('')
```

Comment:
nitpick: Since the email field is a level above the `input[1]` would you mind doing the inverse check like you did in lines 723-725 so that we can avoid false positive `''` check? The biggest issue is that the way were checking for `error = ''` here can also be true if the plugin didn't do anything, so best always check against an error state in this case

Reply:
Once this is addressed I think we're ready to merge :)